.github/workflows/run-unit-tests.yaml

```diff
@@ -28,12 +28,12 @@ jobs:
         uses: actions/checkout@v3
         with:
           ref: ${{ github.ref }}
-          path: ros2_ws/src/panther_ros
+          path: ros2_ws/src/husarion_ugv
```

Comment:
Should, because we specify path where to put whole repo content. IMO unit test should be run on push despite workflow_dispatch and integration test should run on the self-hosted machine and run on PR open and merged.